### ISSUE TITLE
[2.0.0-beta.4] Error when creating new FeedMe

### ISSUE THREAD

**Issue by WHITE-developer** [verbb/navigation#293]
**Description**
When creating a new FeedMe Feed, Craft is throwing a PHP Compile error

**Steps to reproduce**
1. try to create a new FeedMe Feed

**Additional info**
- Plugin version: 2.0.0-beta.4
- Craft version: 4.0.0-RC2
- Multi-site: No

**Additional context**
```
yii\base\ErrorException: Type of verbb\navigation\integrations\NodeFeedMeElement::$element must not be defined (as in class craft\feedme\base\Element) in /var/www/html/vendor/verbb/navigation/src/integrations/NodeFeedMeElement.php:139
Stack trace:
#0 /var/www/html/vendor/yiisoft/yii2/base/ErrorHandler.php(0): craft\web\ErrorHandler->handleFatalError()
#1 /var/www/html/vendor/composer/ClassLoader.php(346): ::Composer\Autoload\includeFile(file: ''/var/www/html/...')
#2 /var/www/html/vendor/craftcms/cms/src/helpers/Component.php(40): Composer\Autoload\ClassLoader->loadClass(class: ''verbb\\\\navigat...')
#3 /var/www/html/vendor/craftcms/cms/src/helpers/Component.php(40): ::class_exists(class: ''verbb\\\\navigat...')
#4 /var/www/html/vendor/craftcms/cms/src/helpers/Component.php(111): craft\helpers\Component::validateComponentClass(class: ''verbb\\\\navigat...', instanceOf: ''craft\\\\feedme\\...', throwException: 'TRUE')
#5 /var/www/html/vendor/craftcms/feed-me/src/services/Elements.php(142): craft\helpers\Component::createComponent(config: '['type' => 'ver...', instanceOf: ''craft\\\\feedme\\...')
#6 /var/www/html/vendor/craftcms/feed-me/src/services/Elements.php(57): craft\feedme\services\Elements->createElement(config: ''verbb\\\\navigat...')
#7 /var/www/html/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\feedme\services\Elements->init()
#8 /var/www/html/vendor/craftcms/cms/src/base/Model.php(78): craft\feedme\services\Elements->__construct(config: '[]')
#9 /var/www/html/vendor/yiisoft/yii2/di/Container.php(411): craft\feedme\services\Elements->__construct(config: '[]')
#10 /var/www/html/vendor/yiisoft/yii2/di/Container.php(411): ReflectionClass->newInstanceArgs(args: '[0 => []]')
#11 /var/www/html/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build(class: ''craft\\\\feedme\\...', params: '[]', config: '[]')
#12 /var/www/html/vendor/yiisoft/yii2/BaseYii.php(345): yii\di\Container->get(class: ''craft\\\\feedme\\...', params: '[]', config: '???')
#13 /var/www/html/vendor/yiisoft/yii2/di/ServiceLocator.php(137): yii\BaseYii::createObject(type: ''craft\\\\feedme\\...', params: '???')
#14 /var/www/html/vendor/yiisoft/yii2/base/Module.php(769): craft\feedme\Plugin->get(id: ''elements'', throwException: 'FALSE')
#15 /var/www/html/vendor/yiisoft/yii2/di/ServiceLocator.php(74): craft\feedme\Plugin->get(id: ''elements'', throwException: '???')
#16 /var/www/html/vendor/craftcms/feed-me/src/controllers/FeedsController.php(68): craft\feedme\Plugin->__get(name: ''elements'')
#17 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): craft\feedme\controllers\FeedsController->actionEditFeed(feedId: 'NULL', feed: 'NULL')
#18 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): ::call_user_func_array:{/var/www/html/vendor/yiisoft/yii2/base/InlineAction.php:57}(callback: '[0 => class cra...', args: '[0 => NULL, 1 =...')
#19 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(params: '[]')
#20 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): craft\feedme\controllers\FeedsController->runAction(id: ''edit-feed'', params: '[]')
#21 /var/www/html/vendor/craftcms/cms/src/web/Application.php(301): craft\web\Application->runAction(route: ''feed-me/feeds/...', params: '[]')
#22 /var/www/html/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction(route: ''feed-me/feeds/...', params: '[]')
#23 /var/www/html/vendor/craftcms/cms/src/web/Application.php(286): craft\web\Application->handleRequest(request: 'class craft\\web...')
#24 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(request: 'class craft\\web...', skipSpecialHandling: '???')
#25 /var/www/html/web/index.php(24): craft\web\Application->run()
#26 {main}

```

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-4 as 2.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by WHITE-developer** [comment]
@engram-design Thanks, fix is working

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.5](https://github.com/verbb/navigation/releases/tag/2.0.0-beta.5)
